redbot/cogs/downloader/downloader.py

```diff
@@ -722,7 +720,7 @@ def __init__(self, bot: Red):
         Use `[p]cog update` to update cogs.
         """
         async with ctx.typing():
-            cogs_to_check = await self._get_cogs_to_check()
+            cogs_to_check, failed = await self._get_cogs_to_check()
```

Comment:
We should probably mention failed repos here too since the repo update happens when running this command too.